packages/near-snap/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@near-wallet-selector/near-snap",
+  "version": "7.0.2",
+  "peerDependencies": {
+    "near-api-js": "^0.44.2"
+  }
```

Comment:
Can you update the version to the latest version of the wallet selector, currently `v7.3.0`

Can you update the range of the peerDependency to this, because now we support `v1.x` of `near-api-js`:

```json
"peerDependencies": {
    "near-api-js": "^0.44.2 || ^1.0.0"
 }
```